### ISSUE TITLE
Collapsible lists remain keyboard accessible when collapsed; need to implement aria-expanded

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2247]
## Steps to reproduce
1. Go to a text that you have annotated
2. Open the Notes panel in the readers banner
3. Use keyboard to navigate to list of notes
4. Navigate to chapter collapsible element
5. Notice that there is no aria-expanded attribute
6. Use space to collapse this item
7. Press tab and notice that focus moves to an invisible element

## Observed Behavior
Collapsible lists do not implement aria-expanded. A non visual user would have no idea that these items are collapsible with out the small chevron icon to help them discover this functionality. When these items are collapsed their child items remain accessible to keyboard navigation. This makes the collapsible functionality useless for keyboard users and is also extremely confusing.

## Expected Behavior
Aria-expanded should be used so that non-visual users are aware of functionality.
When collapsed, the child elements should not be accessible.

## Test Environment
NVDA 2019.1.1
ESR Firefox 60.7.0esr
Windows 1903 (build 18362.145)

**Comment by avaaec** [comment]
Hi! Just noticed that this same problem occurs in the search options for the back end.
## To Reproduce
1. Go to /backend/records for example
2. Navigate to search field
3. press tab three times and notice that your focus is on a combo box that should be hidden

**Comment by zdavis** [comment]
@dananjohnson Could you also address the BE search options in this PR? There's a single component that renders all those options, so hopefully not a particularly difficult fix.

**Comment by dananjohnson** [comment]
@zdavis I've updated the PR.

**Comment by zdavis** [comment]
Thanks, @dananjohnson

**Comment by dananjohnson** [comment]
@zdavis i've found a few other places where this should be addressed. I'll update the PR to address those places too.

**Comment by dananjohnson** [comment]
@zdavis I've updated the PR.

**Comment by avaaec** [comment]
Fix has been verified on edge.manifoldapp! Looking good.
